### DISCUSSION TITLE
The mobile app icon is a bit meaningless

### DISCUSSION THREAD

**Discussion by danddinh** [standardnotes/forum#3615] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes

**Is your feature request related to a problem? Please describe.**
The mobile app icon looks like an empty window, a bit meaningless.

**Describe the solution you'd like**
Should this kind of icon be better: 
![](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTUKe2J3tzAA1TjYD6s6Vuz9AC8qcuvPUlOzw&s)

**Describe alternatives you've considered**
None

**Additional context**
None

**Comment by Donnnno** [comment]
For what it's worth, the [Arcticons icon pack](https://www.arcticons.com/) offers an logo alt icon :-) 

<img width="269" alt="screenshot" src="https://github.com/standardnotes/forum/assets/31142286/0ade28f4-a94a-45a0-859a-9ca6200a3696">
